lib/siilar/client/clients.rb

```diff
@@ -42,6 +46,11 @@ class TagsService < ClientService
 
     class UsersService < ClientService
       include Client::Users
+
```

Comment:
@gcolson The build still fails.
Looks like and `end` is missing here too for this class.